scripts/quickstart.sh

```diff
@@ -24,6 +26,8 @@ ADDRESS=""
 NODE_KEY=""
 INSTANCE_NAME=""
 PLATFORM=""
+ETHBLOCKNUMBER=""
+FUSEBLOCKNUMBER=""
```

Comment:
Actually didn't need these globals all done inside the new functions. I will delete them in the morning :)

Reply:
Cool, so remove them when you can and I'll merge the PR

Reply:
Done :)